src/Tasks/JTask.php

```diff
@@ -69,11 +69,6 @@ public function __construct($params = array())
 		$this->loadConfiguration($params);
 		$this->determineOperatingSystem();
 		$this->determineSourceFolder();
-
-		// Registers the application to run Robo commands
```

Comment:
I sent a fix for this in #30 since it needs to be set before the configuration.  I'll fix that PR to get it merged asap.